src/go/k8s/main.go

```diff
@@ -24,6 +26,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
+const (
+	configuratorContainerImage = "vectorized/configurator"
```

Comment:
should this be explicitly named sth like `UpstreamConfiguratorContainerImage` or `DefaultconfiguratorContainerImage` ?

Reply:
I will change to `DefaultconfiguratorContainerImage`, thank you.